internal/services/compute/managed_disk_resource_test.go

```diff
@@ -475,6 +475,28 @@ func TestAccAzureRMManagedDisk_create_withTrustedLaunchEnabled(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMManagedDisk_update_withBurstingEnabled(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_managed_disk", "test")
+	r := ManagedDiskResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.empty(data),
```

Comment:
can we add a separate test which confirms the Create flow works too, preferably toggling that on/off in the update too

Reply:
I added a test step create_withOnDemandBursting which explicitly set the property to false before upate_withOnDemandBursting so the update will toggle the setting from false to true.
The on-demand bursting can only be turned off after 12 hours after enabling it, so i only tested the off->on scenario